tests/e2e/pact/broker/publish.ts

```diff
@@ -20,5 +22,5 @@ async function publish(): Promise<Array<string>> {
 }
 
 (async () => {
```

Comment:
Do we need a final catch to log any errors? Would we get anything useful from that?

Reply:
I haven't yet come across publishing pacts failing 3 times in a row, the retry here is for dealing with instability which I think comes with hosting the pact broker in heroku

Reply:
:+1: